### PR TITLE
Update to new activity endpoint

### LINKED ISSUE #235: Update to use new activity endpoint
**Is your feature request related to a problem? Please describe.**
Update endpoints where activity have to be updated in the application to use the new activity endpoint

**Describe the solution you'd like**
i.e 
`_api.week.update( _user.id, _weekModel.weekYear, _weekModel.weekNumber, _weekModel).listen((WeekModel weekModel)`
->
`_api.activity.update(_activityModel).listen((ActivityModel activityModel)`

**Describe alternatives you've considered**
None

**Additional context**
This requires https://github.com/aau-giraf/web-api/issues/18 and https://github.com/aau-giraf/api_client/issues/7

### PR DESCRIPTION
**Pull request by boisenme**
Closes #235

### CODE REVIEW

File: lib/blocs/activity_bloc.dart [hunk 1]
```diff
@@ -53,13 +49,11 @@ class ActivityBloc extends BlocBase {
 
   /// Update the weekmodel with the new state.
```
Comment (KVISDAOWNER) [inline lib/blocs/activity_bloc.dart]: kommentaren passer ikke helt mere

File: lib/blocs/activity_bloc.dart [hunk 2]
```diff
@@ -21,14 +20,11 @@ class ActivityBloc extends BlocBase {
 
   final Api _api;
   ActivityModel _activityModel;
-  WeekModel _weekModel;
   UsernameModel _user;
 
   /// Loads the WeekModel, ActivityModel and the GirafUser.
```
Comment (KVISDAOWNER) [inline lib/blocs/activity_bloc.dart]: kommentaren passer ikke helt mere

**Review by KVISDAOWNER — CHANGES_REQUESTED** [review]
Looks good, I have a few request to comments that needs update

File: lib/blocs/activity_bloc.dart (hunk 1 above)
Comment (boisenme) [reply lib/blocs/activity_bloc.dart]: Godt spottet 👯

File: lib/blocs/activity_bloc.dart (hunk 2 above)
Comment (boisenme) [reply lib/blocs/activity_bloc.dart]: Godt spottet 👯

**Review by KVISDAOWNER — APPROVED** [review]
Looks good :D

**Review by MathiasLybech — APPROVED** [review]
PO approved

**Review by andpaacoke — APPROVED** [review]

**Pull request merged**